canonical/memcached-operator
# DPE-40 Implementation of start event

**What needs to be reviewed:**
1. Implementation of _on_start() event
2. Associated unit tests

**What should not be reviewed:**
1. Implementation of _on_config_changed() as this is only a temporary hack to get memcached configured so that it is visible from outside the container

**How to test**
1. `charmcraft pack` and `juju deploy` as usual
2. Use `juju status` to find out assigned IP address, the port is currently 11211
3. Use `telnet <IP_address> 11211
4. Once connected, type `stats` followed by ENTER
5. If you want to try other commands, see [this page](https://techleader.pro/a/90-Accessing-Memcached-from-the-command-line)
6. Once done, type `quit` followed by ENTER
